.pre-commit-config.yaml

```diff
@@ -35,6 +35,14 @@ repos:
       args: [--rcfile=.pylintrc]
       files: ^pymc/
       exclude: (?x)(pymc/_version.py)
+- repo: https://github.com/PyCQA/pydocstyle
+  rev: 6.1.1
+  hooks:
+    - id: pydocstyle
+      args:
+      - --ignore=D100,D101,D102,D103,D104,D105,D107,D200,D202,D203,D204,D205,D209,D212,D213,D301,D400,D401,D403,D413,D415,D417
```

Comment:
The convention arg is missing, we should be using `numpy` as convention (which should remove the contradictory warnings happening all at the same time)

Reply:
Just tried it, but with `--convention` one can't pass `--ignore` and then we'd have hundreds of errors and would need to fix them all in one go (unrealistic..)

Reply:
let's keep ignore then